engine/runtime/src/main/java/org/enso/interpreter/runtime/data/atom/DebugAtomWrapper.java

```diff
@@ -0,0 +1,77 @@
+package org.enso.interpreter.runtime.data.atom;
+
+import com.oracle.truffle.api.interop.InteropLibrary;
+import com.oracle.truffle.api.interop.UnknownIdentifierException;
+import com.oracle.truffle.api.library.CachedLibrary;
+import com.oracle.truffle.api.library.ExportLibrary;
+import com.oracle.truffle.api.library.ExportMessage;
+import com.oracle.truffle.api.nodes.ExplodeLoop;
+import org.enso.interpreter.node.callable.InteropApplicationNode;
+import org.enso.interpreter.runtime.data.EnsoObject;
+
+/**
+ * Wrapper for {@link Atom} that is meant to be used by inspector and the debugger. This wrapper
+ * changes the behavior of some atom interop messages to be more debugger-friendly. For example, it
+ * does not invoke parameter-less instance methods when inspecting the atom. Note that from the
+ * engine and language perspective, there is no difference between a field and parameter-less
+ * instance method.
+ */
+@ExportLibrary(value = InteropLibrary.class, delegateTo = "atom")
+public final class DebugAtomWrapper extends EnsoObject {
```

Comment:
- Can this class be moved to debugger?
- e.g. next to `DebugLocalScope`? 
- then the class wouldn't need to be `public` and yet it could still delegate the messages to `Atom`